app/src/main/java/org/simple/clinic/navigation/v2/Router.kt

```diff
@@ -1,5 +1,6 @@
 package org.simple.clinic.navigation.v2
 
```

Comment:
Can update this to "Add support for saving and restoring the screen backstack"

Reply:
Done.